generator/sbpg/targets/rust.py

```diff
@@ -351,6 +376,20 @@ def __init__(self, msg, package_specs, field):
         if self.options.get("fields", False):
             self.bitfield = get_bitfield(self)
 
+# pattern to capture {{groups}} for enriched message display
+ENRICH_PAT = re.compile("{{([^}]+)}}[.]*", re.S)
+ENRICH_FIELD_PAT = re.compile("@ ([^@]+) @[^\@]*", re.S)
```

Comment:
Be consistent with escaping the `@`.  The final `[^\@]*` is also a bit confusing here I don't think this part is necessary and `@ ([^@]+) @` should work just as well.